bspump/avro/loader.py

```diff
@@ -13,14 +13,9 @@
 
 
 def load_avro_schema(config):
-    schema = config.get('schema')
-    if schema == '':
-        schema_file = config.get('schema_file')
-        if schema_file != '':
-            with open(schema_file, 'r') as fi:
-                schema = json.load(fi)
+    schema_file = config.get('schema_file')
+    if schema_file != '':
```

Comment:
```
 if schema_file == '':
     return None
```

Reply:
@ateska If we don't provide a return value explicitly, then Python will return an implicit 'None' correct?
 Then fastavro.reader, in that case, will set None for 'reader_schema'. Which means it will use a built-in schema.

Reply:
Yes, Python does that but it is less readable than explicit `return None`.